task-list/app/controllers/people_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+  end
+
+  def show
+    id = params[:id]
+    @person = Person.find(id)
+  end
+
+  def show_tasks
+    @person = Person.find(params[:id])
+    @tasks = Person.find(params[:id]).tasks
```

Comment:
Since you already did the `Person.find(params[:id])` on the line above, you don't want to do it again here. That will query the database twice. You should instead use the `@person` variable to retrieve the tasks.